package.json

```diff
@@ -29,7 +29,8 @@
     "clean": "rimraf './dist'",
     "clean:stories": "rimraf './storybook-static'",
     "format": "prettier . --write",
-    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 && npm run lint:styles",
+    "lint:styles": "stylelint 'src/**/*.scss' --fix",
```

Comment:
I revisited my issue on this and it looks like I get that error because src/**/*.scss is wrapped in single quotes. I was able to fix it on my end by escaping the double quotes like this: 
 `stylelint \"src/**/*.scss\" --fix` 
 
 I found the solution here for reference: https://github.com/stylelint/stylelint/issues/4178
 
 Everything looks good to me here after we resolve this 👍